app/datamapper_setup.rb

```diff
@@ -0,0 +1,13 @@
+require 'data_mapper'
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] ||
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.